functional-tests/rollup_params_cfg.py

```diff
@@ -48,3 +48,13 @@ class RollupConfig(BaseModel):
     proof_publish_mode: ProofPublishMode
     max_deposits_in_block: int
     network: str
+
+    # Additional fields that aren't coming from datatool config generation (yet)
+    # and has to be supplied manually.
+    # TODO: extend datatool to return OPERATOR_FEE from bridge-tx-builder/src/constants.rs
```

Comment:
We should open a ticket, or we'll forget about this.

Reply:
Agree, done